velox/exec/Spill.h

```diff
@@ -77,8 +77,6 @@ class SpillFile {
         sortCompareFlags_.size() == numSortingKeys_);
   }
 
-  ~SpillFile();
```

Comment:
Add following note to SpillFile constructor
```
/// NOTE: we don't delete each individual spill file on destructor but do batch deletion by removing the entire directory when the associated query task finishes.
```

Reply:
Is spill only actual for a task?
Can we use it somewhere else?
Better then to add comment for the SpillFile that we don't delete individual files in destructor.

Reply:
I mean the comment clearly say we do batch deletion through directory deletion when a query task finishes.

Reply:
I mean that SpillFile is not necessarily going to be used from a Task, right?
Then saying that Task is going to delete the folder where the file is can raise an eyebrow.